tasks/pre_install_checks.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Gather facts on installed packages
+  package_facts:
+
+- name: Fail when {{ default_firewall_manager }} is not installed
+  fail:
+    msg: '{{ ovirt_engine_setup_firewall_manager }} was chosen as default firewall manager but is not installed'
+  when: >
+    ovirt_engine_setup_firewall_manager == default_firewall_manager and
+    default_firewall_manager not in ansible_facts.packages
```

Comment:
Would need to check if the value is `null` to skip it because of this use-case in README `empty value to skip firewall configuration`

Reply:
I'm afraid I don't understand this comment, could you elaborate?